pkg/keyspace/tso_keyspace_group.go

```diff
@@ -95,8 +96,9 @@ func NewKeyspaceGroupManager(ctx context.Context, store endpoint.KeyspaceGroupSt
 		client:           client,
 		tsoServiceKey:    key,
 		tsoServiceEndKey: clientv3.GetPrefixRangeEnd(key) + "/",
-		policy:           defaultBalancerPolicy,
 		groups:           groups,
+		nodesBalancer:    balancer.GenByPolicy[string](defaultBalancerPolicy),
+		serviceRegistryMap: make(map[string]string),
```

Comment:
Do we still need this map?

Reply:
see https://github.com/tikv/pd/pull/6346